components/menu/demo/radio-menu.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../menu.js';
+			import '../menu-item-radio.js';
+		</script>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page page-title="d2l-menu (radio)">
+
+			<h2>Radio Menu</h2>
+
+			<d2l-demo-snippet>
+				<d2l-menu id="menu" label="Radio Menu" style="max-width:400px;">
+					<d2l-menu-item-radio text="Radio Option 1 with a very long title that will be cut off" value="1"></d2l-menu-item-radio>
+					<d2l-menu-item-radio text="Radio Option 2" value="2"></d2l-menu-item-radio>
+					<d2l-menu-item-radio text="Radio Option 3" value="3"></d2l-menu-item-radio>
+				</d2l-menu>
+			</d2l-demo-snippet>
+
+		</d2l-demo-page>
+
+		<script>
```

Comment:
It would be fine to move this `<script>` inside the `<d2l-demo-snippet>`.  It will get stamped into the document, but it will also show in the formatted code so devs can see the `d2l-menu-item-change` usage.